pkg/registry/certificates/certificates/registry.go

```diff
@@ -56,12 +56,12 @@ func (s *storage) ListCSRs(ctx genericapirequest.Context, options *metainternalv
 }
 
 func (s *storage) CreateCSR(ctx genericapirequest.Context, csr *certificates.CertificateSigningRequest) error {
```

Comment:
file looks dead.  please delete instead.

Reply:
We have plenty of these which are not called. This will be fixed in a follow-up via #54693.